sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql

```diff
@@ -1287,6 +1292,9 @@ aggregates AS (
       STRUCT(ARRAY_CONCAT_AGG(browser_search_content_urlbar_handoff)),
       STRUCT(ARRAY_CONCAT_AGG(browser_search_withads_urlbar_handoff)),
       STRUCT(ARRAY_CONCAT_AGG(browser_search_adclicks_urlbar_handoff)),
+      STRUCT(ARRAY_CONCAT_AGG(browser_search_content_urlbar_persisted)),
+      STRUCT(ARRAY_CONCAT_AGG(browser_search_withads_urlbar_persisted)),
+      STRUCT(ARRAY_CONCAT_AGG(browser_search_adclicks_urlbar_persisted)),
```

Comment:
I think you'll have to move these to the end of the array since lines 1539-1541 offset these fields in order. So these would be moves to lines 1314-1316